### ISSUE TITLE
Error trying to print plugin docstrings at server start-up

### ISSUE THREAD

**Issue by bcanyelles** [alerta/alerta#797]
The server raise an AttributeError at start-up

The error message is: 
`{AttributeError} module 'alerta.app' has no attribute 'logger'`

It is caused by this code fragment:
```python
@add_metaclass(abc.ABCMeta)
class PluginBase:

    def __init__(self, name=None):
        self.name = name or self.__module__
        if self.__doc__:
            from alerta.plugins import app
            app.logger.info('\n{}\n'.format(self.__doc__))
```

To reproduce it you can create a plugin with a docstring and try to start-up the server.

Example:
```python
import logging

logger = logging.getLogger(LOGGER_NAME)

class TestPlugin(PluginBase):
    """
    Example
    """
    def __init__(self):
        logger.info("Configure plugin")
        super(TestPlugin, self).__init__()
```

**Comment by satterly** [comment]
What version of Alerta server? I assume you just upgraded to version 6.6.0? Are you using the docker image? If not, how are you deploying Alerta?

**Comment by satterly** [comment]
Fixed in version 6.6.1 available now. Thanks for reporting the issue.
